style.css

```diff
@@ -696,53 +707,83 @@ body {
 
 @media (max-width: 768px) {
   #contact {
+    background-color: #fff;
     width: 375px;
   }
 
-  .contact-container {
-    background-image: url(image/contact-form-shapes.png);
-  }
-
-  .contact-me div {
-    display: flex;
-    justify-content: center;
+  #contact .contact-case {
+    background-color: #6070ff;
+    padding: 0 0;
   }
 
-  .contact-container .contact-me {
+  .contact-image {
     display: flex;
     justify-content: center;
+    background-image: url(image/contact-form-shapes.png);
+    background-color: #6070ff;
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding: 150px 0;
+    border-radius: 100px 0 0 0;
   }
 
-  .contact-container .para {
+  .contact-container h2 {
     display: flex;
     justify-content: center;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    margin-bottom: 20px;
   }
 
-  .contact-container .para p {
+  .contact-container p {
     width: 327px;
     text-align: center;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #ebebff;
+    margin-bottom: 20px;
   }
 
-  .contact-container form div li {
+  .contact-container form {
     display: flex;
     justify-content: center;
   }
 
-  .contact-container form div li input {
+  .contact-container form ul li {
+    margin-bottom: 20px;
+  }
+
+  .contact-container form input {
     width: 327px;
+    padding: 15px 15px;
+    border-radius: 8px;
+    border: 1px solid #6070ff;
   }
 
-  .contact-container form div li textarea {
+  .contact-container form textarea {
     width: 327px;
+    padding: 15px 15px;
+    border-radius: 8px;
+    border: 1px solid #6070ff;
   }
 
   .contact-container form .btn {
     display: flex;
-    justify-content: center;
+    justify-content: start;
   }
 
-  .contact-container form .btn div {
-    width: 327px;
-    display: inline;
+  .contact-container form .btn button {
+    padding: 10px 15px;
+    border-radius: 8px;
+    border: 1px solid #6070ff;
+    color: #6070ff;
+    transition: all 1s;
+  }
+
+  .contact-container form .btn button:hover {
+    transform: scale(1.2);
   }
 }
```

Comment:
- [ ] Please as was requested by the last reviewer, fix this weird-looking distance between the work section and the headline section, I noticed that you ask about the issue and that it doesn't appear on your screen, the reason for this is that you are using this `vh` size which from screen to screen depending on the height of the screen, if you look at all what you have learned about sizes you can see that we always care about the width you can use `vw` sizes instead, and I would love to tell about this trick you can use this [website](https://web-development.space/tools/px-to-vw/) if you want to use responsive sizes you have to use it in all the website in everything even in the font-size but you should use the website to compare between the width in the design and the port-width in the design and your code you can put px sizes and then you can get `vw` sizes then which will be very responsive.
![image](https://user-images.githubusercontent.com/74744787/215213749-c07ff94f-49f0-4a46-b551-2de19906d38f.png)
![image](https://user-images.githubusercontent.com/74744787/215213959-f580e400-7c13-4be5-a9b4-24db7305c5ed.png)